sdk/src/tx/retryTxSender.ts

```diff
@@ -120,7 +120,7 @@ export class RetryTxSender implements TxSender {
 		opts?: ConfirmOptions
 	): Promise<TxSigAndSlot> {
 		// @ts-ignore
-		tx.sign(additionalSigners.concat(this.provider.wallet.payer));
+		tx.sign((additionalSigners ?? []).concat(this.provider.wallet.payer));
```

Comment:
unrelated but kept seeing in the console that we were trying to concat to undefined